pkg/cluster/internal/providers/nerdctl/provider.go

```diff
@@ -125,7 +125,7 @@ func (p *provider) ListClusters() ([]string, error) {
 		// filter for nodes with the cluster label
 		"--filter", "label="+clusterLabelKey,
 		// format to include the cluster name
-		"--format", fmt.Sprintf(`{{index .Labels "%s"}}`, clusterLabelKey),
+		"--format", fmt.Sprintf(`{{.Label "%s"}}`, clusterLabelKey),
```

Comment:
Do we know if this ever worked? Do we need to version gate it? Or was it just copy-pasted from the docker provider and never worked?

Reply:
Probably this was just copied without testing?

Reply:
This kind of difference should be mentioned in code comment lines

Reply:
it seems podman also already implemented this https://github.com/containers/podman/pull/20958

It is using the old format https://github.com/kubernetes-sigs/kind/blob/f3a83441a7db2ecf3432d973ef9bdcf618215065/pkg/cluster/internal/providers/podman/provider.go#L110-L111

Reply:
We should check the docker driver and probably re-align this.